### ISSUE TITLE
train/loss

### ISSUE THREAD

**Issue by JayjieL** [xuannianz/keras-CenterNet#6]
![image](https://user-images.githubusercontent.com/39478235/69326898-96e61200-0c87-11ea-8b7b-699fb74c7dc1.png)
what's the mean of loss here?

**Comment by xuannianz** [comment]
It's a usual method to define custom loss function in keras. 
Then the name of loss tensor would be `centernet_loss` which corresponds to the loss tensor in the model.
https://github.com/xuannianz/keras-CenterNet/blob/34464328ad1436789c757997619dbadaa8e962e8/models/resnet.py#L171-L172

**Comment by JayjieL** [comment]
pardon me. 
I just don't understand  [ lambda y_true,y_pred : y_pred ]  here. Is that mean this lambda funtion got the return y_pred ?
